### PR TITLE
Mayhem integration

### PR DESCRIPTION
**Pull request by jaketbrown**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build Stage
```
Comment (xansec) [inline mayhem/Dockerfile]: I recommend removing the commented out parts of the Dockerfile.

**Review by xansec — CHANGES_REQUESTED** [review]
There are over 30 added .o and .eo files. Are these necessary? Where did they come from? I recommend removing them from this commit. Please make the requested changes.

File: mayhem/Dockerfile (hunk 1 above)
Comment (jaketbrown) [reply mayhem/Dockerfile]: Done. Regarding the .o and .eo files, I believe the repo maintainer added those, not sure what they're for.

Comment (jaketbrown) [reply mayhem/Dockerfile]: I've cleaned up the .o and .eo files from the repo. They're the product of build the binary used in the mayhemfile.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**